Dockerfile

```
@@ -1,33 +1,37 @@
-    apt-get install -qq --no-install-recommends elasticsearch && \
-    mkdir /usr/src/ebwiki
+# Use Ruby 3.4.2 as base image
+FROM ruby:3.4.2-slim
 
+# Set environment variables
+ENV RAILS_ENV=development
+ENV BUNDLE_PATH=/usr/local/bundle
+ENV BUNDLE_WITHOUT=""
+
+# Install system dependencies
+RUN apt-get update -qq && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)